frontend/src/views/TransferView.vue

```diff
@@ -21,58 +21,74 @@
+            console.log("Fetching from " + url);
+            fetch(url, {
+                headers: {
+                    "X-API-KEY": this.backendApiKey,
+                },
+            })
+                .then((response) => response.json())
+                .then((data) => {
+                    this.catalog = data;
+                })
+                .catch((err) => console.log(err));
+        },
+    },
+    mounted() {
+        let vm = this;
+        vm.getCatalog();
     },
-  },
-  mounted() {
-    let vm = this;
-    vm.getCatalog();
-  },
 };
 </script>
 
 <template>
-  <main>
-    <h1 class="w-full text-center bold text-5xl mb-6 pb-6">
-      View EDC Transfers
-    </h1>
-    <li class="list-none" v-for="offer in catalog">
-      <div
-        class="text-center mx-4 my-8 block p-6 w-full bg-white rounded-lg border border-gray-200 shadow-md hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700"
-      >
-        <div>
-          <h1
-            class="mb-2 text-xl font-bold tracking-tight text-gray-900 dark:text-white"
-          >
-            Transfer {{ offer.id }}
-          </h1>
-          <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
-            State: {{ offer.state }}
-          </h2>
-          <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
-            Type: {{ offer.type }}
-          </h2>
-          <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
-            Asset Id: {{ offer.dataRequest.assetId }}
-          </h2>
-          <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
-            Contract Id: {{ offer.dataRequest.contractId }}
-          </h2>
-        </div>
-      </div>
-    </li>
-  </main>
+    <main>
+        <h1 class="w-full text-center bold text-5xl mb-6 pb-6">
+            View EDC Transfers
+        </h1>
+        <li class="list-none" v-for="item in catalog">
+            <div
+                class="text-center mx-4 my-8 block p-6 w-full bg-white rounded-lg border border-gray-200 shadow-md hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700"
+            >
+                <div>
+                    <h1
+                        class="mb-2 text-xl font-bold tracking-tight text-gray-900 dark:text-white"
+                    >
+                        Transfer {{ item["@id"] }}
+                    </h1>
+                    <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
+                        State: {{ item["edc:correlationId"] }}
+                    </h2>
+                    <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
+                        State: {{ item["edc:state"] }}
+                    </h2>
+                    <h2 class="font-normal text-medium text-gray-700 dark:text-gray-400">
+                        State: {{ item["edc:stateTimestamp"] }}
```

Comment:
Duplicate name "State", Should be "State Timestamp" or "Timestamp".

Could you please also use the same format as in negotiations view for the timestamp? Currently it's just rendered as long number (time since 1970?)